code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -203,28 +202,26 @@
 			var/contained = reagentlist()
 			while(reagents.total_volume)
 				if(ismob(target))
-					trans += reagents.trans_to_mob(target, amount_per_transfer_from_this, CHEM_BLOOD)	
+					trans += reagents.trans_to_mob(target, amount_per_transfer_from_this, CHEM_BLOOD)
 				else
 					trans += reagents.trans_to_obj(target, amount_per_transfer_from_this)
 				update_icon()
 				if(!reagents.total_volume || !do_after(user,cycle_time,target))
 					break
-			
+
 			if (reagents.total_volume <= 0 && mode == SYRINGE_INJECT)
 				mode = SYRINGE_DRAW
 				update_icon()
 
 			if(trans)
 				to_chat(user, "<span class='notice'>You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units.</span>")
-				if(ismob(target))
-					add_attack_logs(user,target,"Injected with [src.name] containing [contained], trasferred [trans] units")
+				add_attack_logs(user,target,"Injected with [src.name] containing [contained], trasferred [trans] units")
 			else
 				to_chat(user, "<span class='notice'>The syringe is empty.</span>")
 
-			dirty(target,affected) //VOREStation Add
 
 	return
-/* VOREStation Edit - See syringes_vr.dm
+
 /obj/item/weapon/reagent_containers/syringe/update_icon()

```

Comment:
Why do you want to revert to the old update_icon? The new one is far more efficient and actually makes the new icons work. Did you test this...? Even if you did, "no", you can't use overlays.Cut() and stuff anymore, you need to use the functions so things don't break.